runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -2721,6 +2721,28 @@ done:;
 		return EXECUTE_BYTECODE;
 	}
 
+	/* java.lang.Class: public native Class<?> arrayType(); */
+	VMINLINE VM_BytecodeAction
+	inlClassArrayType(REGISTER_ARGS_LIST)
+	{
+		VM_BytecodeAction rc = EXECUTE_BYTECODE;
+		j9object_t arrayObject = NULL;
```

Comment:
Local can be better scoped, or removed entirely (assigned and used only on the very next line).